src/status_im/multiaccounts/login/core.cljs

```diff
@@ -213,7 +213,6 @@
                                              :mailserver-ranges {}
                                              :mailserver-topics {}
                                              :default-mailserver true})
```

Comment:
this is not enough, if you removed `:multiaccount/recover`, `recovering-multiaccount?` doesn't work anymore and you can't recover an account properly

Reply:
Ah, thanks, will fix.

Reply:
Done.